consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeCoordinator.java

```diff
@@ -79,7 +79,7 @@ public class MergeCoordinator implements MergeMiningCoordinator, BadChainListene
    */
   private static final double TRY_FILL_BLOCK = 1.0;
 
-  private static final long DEFAULT_TARGET_GAS_LIMIT = 30000000L;
+  private static final long DEFAULT_TARGET_GAS_LIMIT = 36_000_000L;
```

Comment:
do we not also want to use the testnet check and different defaults [here](https://github.com/hyperledger/besu/pull/8249/files#diff-6da2d65dd7a0564527e8ce3ad7c4482f599fb2af30e37316c47854e5aed2e977R137) and [here](https://github.com/hyperledger/besu/pull/8249/files#diff-6da2d65dd7a0564527e8ce3ad7c4482f599fb2af30e37316c47854e5aed2e977R183)?